src/typings.ts

```diff
@@ -152,17 +333,26 @@ export interface RequestLoginLinkOptions extends ConfigsOptions {
 }
 
 export interface TokenInfo {
+  /** token issuer */
   iss: string;
+  /** token cretaion time */
   iat: number;
+  /** token expiry */
   exp: number;
+  /** token audience(s) */
   aud: string[];
+  /** token subject - the moneytree ide fo the user */
   sub: null | string;
   scope: string;
   client_id: null | string;
+  /** application related information */
   app: null | {
+    /** Application name */
     name: string;
+    /** @hidden */
     is_mt: boolean;
   };
+  /** User related informatino */
```

Comment:
Small typo `informatino`
```suggestion
  /** User related information */
```